components/sup/src/event.rs

```diff
@@ -177,6 +179,16 @@ pub fn service_stopped(service: &Service) {
     }
 }
 
+/// Send an event at the start of a Service update.
+pub fn service_update_started(service: &Service, update: &PackageIdent) {
+    if stream_initialized() {
+        publish(ServiceUpdateStartedEvent { event_metadata:       None,
+                                            service_metadata:
+                                                Some(service.to_service_metadata()),
+                                            update_package_ident: format!("{}", update.clone()), });
```

Comment:
This would be more idiomatically rendered as `update.to_string()`, rather than going through the `format!` macro.

I thought Clippy would have caught this with [useless_format](https://rust-lang.github.io/rust-clippy/master/#useless_format), but now I see that only would apply if `update` was a `String`.

Reply:
👍